index.js

```diff
@@ -209,7 +217,7 @@ const template = (data, name) => {
 };
 
 const fileName = (data, name) => {
-  const createdDate = new Date(data.created_at);
+  const createdDate = new Date(data.start_time);
```

Comment:
This will break at other activities. Can we do it in a way to handle both `created_at` & `start_time`?

Reply:
We could add a condition on the `start_time` variable, but I don't see how it will break for other activities.
It seems that `start_time` exists in all session files, with/without live tracking, with/without GPS path. For live tracked sessions, `created_at` and `start_time` have the same value.
In addition, it's already used in your code : https://github.com/glennreyes/runtastic-gpx/blob/master/index.js#L116

Reply:
Let's do `created_at || start_time` here since we cannot assume that start_time will always equal the created date. So we fallback to `start_time` if the creation date doesn't exist.